src/priceFeeder/substrate/feeder.ts

```diff
@@ -3,29 +3,33 @@ import { WsProvider, HttpProvider } from '@polkadot/rpc-provider';
 import Keyring from '@polkadot/keyring';
 import { KeyringPair } from '@polkadot/keyring/types';
 import { stringToU8a } from '@polkadot/util';
-import BigNumber from 'bn.js';
+import BN from 'bignumber.js';
```

Comment:
why use bignumber.js over BN.js? I think polkadot.js is using BN.js?

Reply:
bn.js only deals with integers, and it's a natural choice for libs with only integers like polkadot.js and web3.js.

But for oracle server, it's more receiving floating point price string from api and convert it to int. bn.js doesn't handle this scenario.